### ISSUE TITLE
new custom command to update the service-URL in the database

### ISSUE THREAD

**Issue by CliffyPrime** [BookStackApp/BookStack#1225]
I would love to see a new custom command like these:
https://www.bookstackapp.com/docs/admin/commands/

The new command should be able to update each and every occurance of the BookStack-URL in every table and column of the database.

This means it would have to accept two parameters:
1. the old, possible invalid URL
2. the new, currently working URL

example:
`php artisan bookstack:update-url http://unsecure.old.link:8080 https://bookstack.new.url`

Reason:
As (internal) Links and images are stored with the URL that was used when uploading the image or creating a link, these links might get broken if you migrate your BookStack-Instance to a new URL. This results in:
- internal links not working
- pictures not being displayed

Manually fixing these things via direct manual SQL-database editing might not be perfect for everyone as it's really easy to mess up, so a "ready to go"-command would be a blessing.

**Comment by lieszkol** [comment]
+1
I've actually written a [short blog entry](https://kb.zensoft.hu/migrating-your-bookstack-app-to-a-new-domain/) on this topic...

**Comment by ssddanbrown** [comment]
This feature is now in the `master` branch, ready to be part of the next release (v0.28). The docs will be updated with this new command upon release.

**Comment by crainsaw** [comment]
For those stumbling upon this feature request: The command is implemented exactly as proposed and documented on [https://www.bookstackapp.com/docs/admin/commands/](https://www.bookstackapp.com/docs/admin/commands/).

**Comment by shortydutchie** [comment]
TLDR;

Example: old url to new url
`php artisan bookstack:update-url http://docs.example.com https://demo.bookstackapp.com `

Afterwards - clear cache with
`php artisan cache:clear `
